src/Devices/Artemis.Plugins.Devices.Wooting/DataModels/WootingDataModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Artemis.Core.Modules;
+using System;
+
+namespace Artemis.Plugins.Devices.Wooting;
+
+public class WootingDataModel : DataModel
+{
+
```

Comment:
It might be a good idea to add a comment explaining why this is empty (I inferred it's because it is using dynamic childs only)